app/views/cloud_tenant/show.html.haml

```diff
@@ -20,5 +20,5 @@
       = render :partial => "layouts/tl_show"
       :javascript
         var miq_after_onload = "miqAsyncAjax('#{ url_for(:action => @ajax_action, :id => @record.id)}');"
-    - else
+    - elsif @showtype == "main"
       = render :partial => @showtype
```

Comment:
@h-kataria I think Brakeman wants `@showtype` removed from the `render` statement.  So, probably,
```ruby
= render :partial => "main"
```